temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
+    with open(filepath, "w") as output_file:
+        if file_type == "txt":
+            output_file.write("")
+        elif file_type == "json":
+            output_file.write(json.dumps({}))
+        else:
+            log.error("ERROR: Invalid file type")
```

Comment:
**Suggestion:** Allow the user to specify a custom reset value for the file. [enhancement]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
def reset_files(filepath, file_type="txt", reset_value=""):
    with open(filepath, "w") as output_file:
        if file_type == "txt":
            output_file.write(reset_value)
        elif file_type == "json":
            output_file.write(json.dumps(reset_value) if reset_value else json.dumps({}))
        else:
            log.error("ERROR: Invalid file type")
```